### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by robgpeak** [robgpeak/CL-WebExtension#31]
@AddisonFreeman please look into the application and see why it is not working

<img width="476" alt="screen shot 2018-11-19 at 12 13 21 pm" src="https://user-images.githubusercontent.com/28393235/48723433-a0f8af00-ebf4-11e8-911d-28c8cde79cf5.png">

**Comment by AddisonFreeman** [comment]
It's trying to pull from uber.complinks.co/api/v1/getUserDetail with an
error message of

1 Failed to load resource: net::ERR_NAME_NOT_RESOLVED

That endpoint doesn't seem to be working right.

**Comment by robgpeak** [comment]
But why would it even be trying to pull from that "uber" cname if the user is not logged in there?

**Comment by AddisonFreeman** [comment]
It has to check all the domain to see which ones it’s logged into. The most
recent login will be the one that appears.

**Comment by AddisonFreeman** [comment]
I can fix it so that it doesn't error on a bad endpoint.

**Comment by robgpeak** [comment]
I just removed the cname record, but it is probably a good idea to fix it on your side as well.

**Comment by robgpeak** [comment]
@cgladue I just tried deleting the uber property but was not able to...  I was only testing something, but it did not work.
<img width="1169" alt="screen shot 2018-11-19 at 1 22 50 pm" src="https://user-images.githubusercontent.com/28393235/48726768-5b40e400-ebfe-11e8-8903-3e0f8c22aa55.png">

**Comment by robgpeak** [comment]
@AddisonFreeman were you able to fix it?  If so, would I have to reload the extension to make it work?

**Comment by AddisonFreeman** [comment]
Yes, you would have to reload it. I spent some time trying to solve the
issue of a bad domain in the list but I'm stuck in that you can't find out
if the url is bad before it gives a critical error. In short we have the
know the domain is good before trying it.

I can remove the uber subdomain in the extension as a patch while the
backend catches up but you'd have to load it another time if you ever want
to use that subdomain in production again.

Are you not able to change the value in the back end?

**Comment by robgpeak** [comment]
Chad was able to remove it and everything is back to normal.  I will keep this in mind if I make any other changes.  Thanks and have a Happy Thanksgiving!

**Comment by AddisonFreeman** [comment]
Let me know before you want to test new stuff out again and I'll give you a
sanity check that it will work.

Take care,
